meters/impl/abb.go

```diff
@@ -95,15 +112,26 @@ func (p *ABBProducer) Produce() (res []Operation) {
 		CurrentL1, CurrentL2, CurrentL3,
 		Power, PowerL1, PowerL2, PowerL3,
 	} {
-		res = append(res, p.snip32(op, 100))
+		res = append(res, p.snip32(op, 10))
 	}
 
 	for _, op := range []Measurement{
-		Cosphi, CosphiL1, CosphiL2, CosphiL3,
 		Frequency,
+	} {
+		res = append(res, p.snip16(op,100))
+	}
+
+	for _, op := range []Measurement{
+		Cosphi, CosphiL1, CosphiL2, CosphiL3,
```

Comment:
Imho other meters return between -/+100 here, to its 0,001 (1000) times 100 which makes division by 10 for keeping the results in sync with other meters? 

please let me know.

Reply:
i did not change this because i cannot verify this values. maybe they are still wrong

Reply:
Would you mind updating to div 10 then? You can see the output in verbose mode and check if ita ok.

Reply:
Updated to div 10